web/thread_page.jsp

```diff
@@ -0,0 +1,68 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" type="text/css" href="deptstyle.css">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <ul>
+        <li><a class="active" href="index.html">Home</a></li>
+        <li><a href="depts.html">Department/Classes</a></li>
+        <li><a href="login.html">Login/Sign Up</a></li>
+        <li><a href="buysell.jsp">Buy/Sell</a></li>
+        <li><a href="contact.html">Contact</a></li>
+    </ul>
+    <%@page import = "java.sql.*" %>
+    <%
+        Class.forName("com.mysql.jdbc.Driver");
+        Connection con = DriverManager.getConnection("jdbc:mysql://localhost:3306/test_db", "root", "BlackCat13");
+        Statement st = con.createStatement();
+        ResultSet rs;
+        rs = st.executeQuery("SELECT * FROM forum WHERE threadid = '" + request.getParameter("id") + "'");
```

Comment:
@rsingh01 thread_page.jsp is basically just a hollow skeleton. It is fleshed out using the id parameter we passed to it from the buysell.jsp page. Here we retrieve the id value using `request.getParameter("id")`. Now that we know which thread we are trying to print on this page, we use the id to retrieve it using `st.executeQuery("SELECT * FROM forum WHERE threadid = '" + request.getParameter("id") + "'")`. Now that we have this rs object (it is just a set that holds the one thread object we need), we can fill the entire page because we now have the thread in question. The page is completely generic.